target.py

```diff
@@ -30,20 +30,7 @@ def __init__(self):
         self.points = target_attr.points
         self.rect = self.image.get_rect()
         #location
-        self.rect.x = random.randint(0, DISPLAY_SIZE[0] - target_attr.width - 20) #-20 for edge buffer
-        self.rect.y = random.randint(0, DISPLAY_SIZE[1] - target_attr.height - 20) #-20 for edge buffer
+        self.rect.x = random.randint(50, DISPLAY_SIZE[0] - (target_attr.width +50)) #-20 for edge buffer
+        self.rect.y = random.randint(50, DISPLAY_SIZE[1] - (target_attr.height +50)) #-20 for edge buffer
```

Comment:
Maybe we should make the 50s a constant?  Also, the comment doesn't match the code anymore.

Reply:
I decided to change it to 50 because there was an instance where the target was unshootable from a horizontal angle. The only reason I didn't add this as a constant is because there is a comment but maybe we should so it can easily be changed in the future